test/ctr_seccomp.bats

```diff
@@ -184,7 +184,7 @@ function teardown() {
 
 # 6. test running with ctr docker/default
 # test that we cannot run with a syscall blocked by the default seccomp profile
-@test "ctr seccomp profiles runtime/default" {
+@test "ctr seccomp profiles docker/default" {
```

Comment:
Why did you have to change the name of the test here?

Reply:
This test name is a duplicate with test number 2 and bats now complains about this.

Reply:
Ok thanks for the clarification!

Reply:
It should be noted that this is intended to be fixed at some point in the future, but it has never actually worked as-expected (the latest-defined test would always be executed).

Reply:
The fun part is that this test never has been executed, but that should be fixed now.